libs/langchain/langchain/chat_models/base.py

```diff
@@ -55,19 +70,37 @@ def init_chat_model(
                 - gemini... -> google_vertexai
                 - command... -> cohere
                 - accounts/fireworks... -> fireworks
+        configurable_fields: Which model parameters are configurable. If
+            - None: No configurable fields.
+            - "any": All fields are configurable.
```

Comment:
Could you add a security note and recommend specifying an explicit list of allowed parameters. If you want you could add the example of the api_url being controllable